.travis.yml

```diff
@@ -4,6 +4,10 @@ jdk:
   - oraclejdk7
   - oraclejdk8
 
+script:
+  - mvn clean -Pstrict compile test-compile -B
```

Comment:
clean shouldn't be required. Also travis executes the install phase before running tests, so we should probably preserve that.